account-decoder/src/parse_account_data.rs

```diff
@@ -54,6 +54,7 @@ pub enum ParseAccountError {
 pub struct ParsedAccount {
     pub program: String,
     pub parsed: Value,
+    pub space: u64,
```

Comment:
I think u32 is sufficient

Reply:
`usize`? Would save the casts.
I only went for u64 to be consistent with `system_instruction::create_account`